lib/src/ConfigLoader.cc

```diff
@@ -557,6 +557,7 @@ static void loadRedisClients(const Json::Value &redisClients)
     for (auto const &client : redisClients)
     {
         auto host = client.get("host", "127.0.0.1").asString();
+        auto hostIp = drogon::utils::resolveHost(host)[0].toIp();
```

Comment:
The resolveHost method may return an empty result that causes this line crashed.

Reply:
@an-tao Fixed it.
The other problem is that it's not compiling on windows and I don't know how to fix that.